README.md

```diff
@@ -59,23 +59,50 @@ The library comprises two packages:
 1. Create a custom layout template for your pages using `hmrcLayout` as per the section
       [HMRC layout](#hmrc-layout)
 
-1.  Optionally, add `TwirlKeys.templateImports` in `build.sbt`:
-    ```sbt
-        TwirlKeys.templateImports ++= Seq(
-          "uk.gov.hmrc.govukfrontend.views.html.components._",
-          "uk.gov.hmrc.govukfrontend.views.html.helpers._",
-          "uk.gov.hmrc.hmrcfrontend.views.html.components._",
-          "uk.gov.hmrc.hmrcfrontend.views.html.helpers._"
-        )
-    ```
-
-    Adding this removes the need for `@import` statements in your Twirl templates.
-    If you prefer not to use this mechanism, import the components in any template that uses them as
-    follows:
-
-    ```scala
-    @import uk.gov.hmrc.hmrcfrontend.views.html.components._
-    ```
+1. Then to use our components and helpers in your templates, you will need to import the component or helper from its
+   package `uk.gov.hmrc.(govuk|hmrc)frontend.views.html.(components|helpers)`.
+
+   For most components the parameters that can be provided are encapsulated in a viewmodel, comprised of case classes
+   that live within a subpackage of `uk.gov.hmrc(govuk|hmrc)frontend.views.viewmodels` and are aliased for use
+   under `uk.gov.hmrc(govuk|hmrc)frontend.views.html.components`.
+
+   ```scala
+   /*
+   Import all components, helpers, viewmodels, and implicits. Most succinct import method, but may require additional 
+   imports to resolve ambiguous import compilation errors for some viewmodels. Will also cause unused import warnings.
+   */
+   @import uk.gov.hmrc.hmrcfrontend.views.html.components._
+   @import uk.gov.hmrc.hmrcfrontend.views.html.helpers._
+   @import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._                  
+
+   /*
+   Import specific components and viewmodels, and all implicits. Avoids possibility of ambiguous import compilation 
+   errors and unused import warnings.
+   */
+   @import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcTimeline      /* component */
+   @import uk.gov.hmrc.hmrcfrontend.views.html.components.{Timeline, Event} /* viewmodel case classes */
+   @import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+
+   @this(hmrcTimeline: HmrcTimeline)
+
+   @()
+
+   @hmrcTimeline(Timeline(
+     events = Seq(
+       Event(
+         title = "Event 1",
+         time = "14 July 2020",
+         datetime = Some("2020-07-15"),
+         content = "Example content"
+       )
+     )
+   ))
+   ```
```

Comment:
As discussed, might be more useful to teams to use a more widely used component, for example `CharacterCount` from `play-frontend-hmrc`. Additionally, if using a form input component, then can demonstrate the use of the implicit `withFormField` method, as currently the import of the implicits doesn't bring anything in that actually gets used.